compatibility.bs

```diff
@@ -1055,11 +1057,61 @@ On mobile platforms, including smaller iPad form factors
   </tbody>
 </table>
 
+
+<h2 id="removal">Deprecation and Removal</h2>
+
+<em>This section is non-normative.</em>
+
+While the web platform stives for backwards-compatibility, there are occasions which require the
+outright removal or discouraged use of some features. To support making these changes
+in the least disruptive and most consistent way for developers, we document them here. This is most
+beneficial as a decalration of intent by browser engines to act together in a well reasoned way.
+
+<h3 id="removal-features">Feature List</h3>
+
+<table>
+  <thead>
+    <tr>
+      <th><dfn for="deprecated feature list">Feature</dfn></th>
+      <th><dfn for="deprecated feature list">Deprecated from</dfn></th>
+      <th><dfn for="deprecated feature list">Removal</dfn></th>
+      <th><dfn for="deprecated feature list">Replacement</dfn></th>
+      <th><dfn for="deprecated feature list">Justification</dfn></th>
+      <th><dfn for="deprecated feature list">Automated Tools</dfn></th>
+    <tr>
+  </thead>
+  <tbody>
+    <tr>
+    </tr>
+  </tbody>
+</table>
+
+<h4 id="removal-features-process">Process</h4>
+
+Anyone can make an addition to the list by editing this specification.
+However, because some content describes the future plans of browser engines, the author can use
+their discretion in what reviewers are appropriate for a given change to the list.
+[[#removal-support]] is provided to give a place to link to long-form content within this
+document for list entries.
+
+Each entry can include:
+
+<ul>
+  <li>the [=deprecated feature list/Feature | feature or behavior=] that is being removed or deprecated.
+  <li>what browser engines have [=deprecated feature list/Deprecated from | deprecated =] the feature or behavior, discouraging its further use.
+  <li>when the feature will be, or has been, [=deprecated feature list/Removal | removed =] from various browser engines, preventing its use by default.
+  <li>what developers should use to achieve the same functional goals, as a [=deprecated feature list/Replacement | replacement =] for the old feature.
+  <li>[=deprecated feature list/Justification=] for the removal or deprecation, including any tradeoffs considered and steps taken to minimize disruption to developers.
```

Comment:
This feels like it could span anywhere from a sentence to a multi-page document... that may get awkward and unreadable in an HTML table. Maybe a link to a different resource like an Intent to Deprecate from blink-dev or gecko-dev, or a WebKit blog post, etc? Or just a design doc or spec issue. 🤷

Reply:
I was imagining a link to a dedicated subsection of Supporting Information. Spec issue or ItD links would also be fine. Honestly, just having a paragraph or two where the PR author commits to "why this needs to happen and what I did to make it hurt devs less" is my goal here.